docs/resources/Application.md

```diff
@@ -160,7 +162,7 @@ Apps that support the user installation context are visible across all of an aut
 
 By default, newly-created apps only support installation to guilds.
 
-You can update which installation contexts your app supports in your [app's settings](https://discord.com/developers/applications). On the **Installation** page under the **Installation Contexts** section, you can select the installation contexts your app supports.
+You can update which installation contexts your app supports in your [app's settings](https://discord.com/developers/applications). On the **Installation** page under the **Authorization Methods** section, you can select the installation contexts your app supports.
```

Comment:
Are these changes here and below intended?  User installable apps are no longer in preview, but thinking maybe this just needs a rebase.

Reply:
Yeah i need rebase this